freqtrade/freqai/RL/BaseReinforcementLearningModel.py

```diff
@@ -320,6 +320,22 @@ def build_ohlc_price_dataframes(self, data_dictionary: dict,
 
         return prices_train, prices_test
 
+    def model_exists(self, dk: FreqaiDataKitchen) -> bool:
+        """
+        Given a pair and path, check if a model already exists
+        :param pair: pair e.g. BTC/USD
+        :param path: path to model
+        :return:
+        :boolean: whether the model file exists or not.
+        """
+        path_to_modelfile = Path(dk.data_path / f"{dk.model_filename}_model.zip")
```

Comment:
@robcaulk would it make sense to have the extension dynamic / in a variable someplace? (.zip for RL models, .joblib for joblib models, .h5 for keras models (where the same thing would be broken, too)) ? 

That'd avoid overwriting / duplicating a whole function just because 3-5 letters are different ...

Reply:
Please see #8182 for a suggested approach that avoids duplicate code.